tools/autoware-interfaces/README.md

```diff
@@ -0,0 +1,30 @@
+# Document generation for autoware interfaces
+
+Some of the documents under the `docs/design/autoware-interfaces` are generated by scripts.
+If you would like to update these documents, please follow the steps below.
+
+## Usage
+
+Create a workspace anywhere and source the dependent message packages.
+
+```bash
+mkdir workspace
+cd workspace
+git clone https://github.com/autowarefoundation/autoware_adapi_msgs.git
+colcon build
+source install/setup.bash
```

Comment:
```suggestion
mkdir -p workspace/src
cd workspace/src
git clone https://github.com/autowarefoundation/autoware_adapi_msgs.git
cd ..
colcon build
source install/setup.bash
```

Reply:
Should it be like this? Or is your version correct?

Reply:
Thanks! It worked without `src` but your version is more preferable.